acados/ocp_nlp/ocp_nlp_constraints_bgh.c

```diff
@@ -1644,6 +1653,27 @@ void ocp_nlp_constraints_bgh_update_qp_vectors(void *config_, void *dims_, void
     // fun[2*ni : 2*(ni+ns)] = - slack + slack_bounds
     blasfeo_daxpy(2*ns, -1.0, memory->ux, nu+nx, &model->d, 2*nb+2*ng+2*nh, &memory->fun, 2*nb+2*ng+2*nh);
 
+    // Set dmask for QP: 0 means unconstrained.
+    for (int i = 0; i < nb+ng+nh; i++)
+    {
+        if (BLASFEO_DVECEL(&model->d, i) <= -ACADOS_INFTY)
```

Comment:
I would go for exactly == , never know if a user wants to solve a badly scaled problem on purpose.
At least exactly == decreases likelihood of removing some desired constraint by chance.

Reply:
Unless there are reasons why it should not hold exactly == with a constraint that was set to ACADOS_INFTY.